Makefile

```diff
@@ -43,23 +43,32 @@ vet: ## Apply go vet to all go files
 build-e2e:
 	go test -c -o bin/e2e github.com/openshift/cluster-api-actuator-pkg/pkg/e2e
 
-.PHONY: test-e2e
-test-e2e: ## Run openshift specific e2e test
+define test =
 	go test -timeout 90m \
```

Comment:
when I run this the command is ignored. Do we need `=`? https://ftp.gnu.org/old-gnu/Manuals/make/html_node/make_59.html#SEC63

```make --version
GNU Make 3.81
Copyright (C) 2006  Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.

This program built for i386-apple-darwin11.3.0
```

Reply:
The command runs for me as it is.